packages/components/steps/src/steps.ts

```diff
@@ -0,0 +1,5 @@
+import { InjectionKey,Slot } from 'vue'
+import type { StepsComponent } from './types'
+
+export const stepsContextInjectionKey: InjectionKey<StepsComponent> = Symbol()
+export const progressDotSlotKey: InjectionKey<Slot> = Symbol()
```

Comment:
```suggestion
export const stepsToken: InjectionKey<{props:StepsProps, progressDot:Slot }> = Symbol()
```
另外这个文件名字也改成 token.ts 吧。